src/applications/personalization/dashboard/utils/helpers.js

```diff
@@ -1,9 +1,21 @@
 import React from 'react';
-import _ from 'lodash';
 import moment from 'moment';
 
-import environment from '../../../platform/utilities/environment';
-import { apiRequest as commonApiClient } from '../../../platform/utilities/api';
+import environment from '../../../../platform/utilities/environment';
+import { apiRequest as commonApiClient } from '../../../../platform/utilities/api';
+
+export function formatDate(date, options = {}) {
```

Comment:
There's also a `formattedDate` in this same module. I'll leave it to @rianfowler to decide whether this `formatDate` be removed in favor of the former when it comes to usage in dashboard.